plasma_framework/contracts/src/exits/payment/controllers/PaymentStartStandardExit.sol

```diff
@@ -43,7 +43,9 @@ library PaymentStartStandardExit {
 
     event ExitStarted(
         address indexed owner,
-        uint168 exitId
+        uint168 exitId,
+        uint256 utxoPos,
+        bytes rlpOutputTx
```

Comment:
hmmm, do we need `rlpOutputTx `? I don't think we do. I thin it's only `utxoPos `

Reply:
https://github.com/omgnetwork/elixir-omg/blob/306246168240d90f9f4cad63092a8f5ea243ea57/apps/omg_watcher/lib/omg_watcher/exit_processor.ex#L717

Reply:
`rlpOutputTx` gets renamed into:
```
{"outputTxInclusionProof", :output_tx_inclusion_proof},
      {"rlpOutputTx", :output_tx},
      {"utxoPos", :utxo_pos}
```

so it's `output_tx`.

Reply:
We need it in our current code @Ino. Check `fields.ex` and exit processor core. We could work around that by getting the tx from rocksdb storage.

Reply:
yeah fileds is a mapping module, it's not really *used* anywhere.

Reply:
@InoMurko how about here:   https://github.com/omgnetwork/elixir-omg/blob/a7b4188c938390b13d069f40d941507ca79faca3/apps/omg_watcher/lib/omg_watcher/exit_processor/exit_info.ex#L66
Isn't it the exiting tx bytes from call data?
Called here: https://github.com/omgnetwork/elixir-omg/blob/a7b4188c938390b13d069f40d941507ca79faca3/apps/omg_watcher/lib/omg_watcher/exit_processor/core.ex#L188